website/source/docs/concepts/tokens.html.md

```diff
@@ -42,14 +42,53 @@ Finally, the `-policy` flag can be used to set the policies associated
 with the token. Learn more about policies on the
 [policies concepts](/docs/concepts/policies.html) page.
 
+<a name="ttls-and-leases" />
 ## Token Time-To-Live and Leases
 
-Every non-root token has a time-to-live (TTL) associated with it.  After the
-TTL is up, the token will no longer function. In addition to no longer
-functioning, Vault will revoke it.
+Every non-root token has a time-to-live (TTL) associated with it, which is
+the number of seconds the token is valid for, _measured from the moment it is
+created_. After the TTL is up, the token will no longer function, and
```

Comment:
> Under the hood there isn't any real TTL being stored; .... Seconds from issue is really only used to calculate the total duration that has passed since issuance and compare that to the max TTL

Gotcha, when I mentioned `Under the hood under the hood TTL represents seconds from issue` I was referring to this [TTL field in `LeaseOptions`](https://github.com/hashicorp/vault/blob/master/logical/lease.go#L10). Was this field introduced when the new TTL terminology was introduced, or is it something that's had to stay around for compatibility reasons?

Do you have any guidance about how I should describe what "Max Lease TTL" is capping? Should I say that tokens can only be "leased" for up to a certain amount of time,  or would that reignite the lease vs ttl confusion?

Thanks again for being so patient & thorough with your explanations, it's really helpful. I didn't realise `creation_ttl` was being stored as well.